docs/DeveloperGuide.md

```diff
@@ -51,30 +51,63 @@ Step 5. The string at the 0th index is then used in a switch statement, where ea
         
 ### Loading Data
 
-Data is loaded automatically from JSON files in the data folder. This is mainly facilitated through the `TopicsStorage` class which handles accessing the file as well as converting from JSON into `Topic`, `Question` and `Option` objects.
+Data is loaded automatically from JSON files in the data folder. This is mainly facilitated through the `TopicsStorage` 
+class which handles accessing the file as well as converting from JSON into `Topic`, `Question` and `Option` objects.
 
 ![TopicsStorage Class Diagram](./images/TopicsStorage.png)
 
 Given below is an example usage scenario of loading in two topics with two questions each.
 
-When the user launches the app, the main program will initialize a `TopicsStorage` object and call the `load` method which will return a `TopicList` object. The following sequence diagram shows how the load operation works:
+When the user launches the app, the main program will initialize a `TopicsStorage` object and call the `load` method 
+which will return a `TopicList` object. The following sequence diagram shows how the load operation works:
 
 ![TopicsStorage load](./images/TopicsStorage_load.png)
 
+### Class Component 
+
+#### Option and OptionList Class 
+
+The `Option` and `OptionList` classes implements the `Displayable` and `DisplayableList` interfaces respectively. 
+The `Option` object stores one option of a question while the `OptionList` object stores all 4 options of the same 
+question. The class diagram below illustrates the structure of both classes. 
+
+![Option_and_OptionList_Class](./images/Option.png)
```

Comment:
Looks good! However, should the multiplicity for the association between OptionList and Displayable be 4 instead of *?

Reply:
Oh ok, I think it's 4! Thank you for spotting it!

Reply:
You can take out add, delete and the empty constructor as I think @joelngyx is working on removing them from our code

Reply:
Updated! Thank you!